web/public/routes/home/home.css

```diff
@@ -52,6 +55,8 @@
 
 .image-box {
   margin-bottom: 20px;
+  padding-bottom: 20px;
+  border-bottom: 1px solid #eaeaea;
```

Comment:
@rvennam 

"border-box" is an existing class defined in darkvision-light/dark.css

you can define image-box in home.css as:
```
border-bottom: 1px solid;
```
without the color then you can add the class "border-box" to the element that has image-box already.

And if the color of the border-box is not what you want, then fine, you add a new "image-box" class in darkvision-light/dark that has only the color definition for the border:
```
border-color: #eaeaea;
```

This way when darkvision.css, home.css and darkvision-light/dark are all put together the css classes are merged and you get your colored border definition.

Look for references to "border-box" in the html code.

darkvision.css, home.css, video.css -> all but colors
darkvision-light/dark -> only colors.